WARNING: No route found for IPv6 destination :: (no default route?)
Traceback (most recent call last):
  File "list_opcodes.py", line 292, in <module>
    main( )
  File "list_opcodes.py", line 276, in main
    records = find_dates(stream)
  File "list_opcodes.py", line 223, in find_dates
    assert datetime is not None, "\n%s" % lib.hexdump(bolus)
AssertionError: 
0000   0x6c 0x03 0x80 0x05 0x0c 0x00 0xe8 0x00    l.......
0008   0x00 0x00 0x00 0x03 0x36 0x03 0x36 0x64    ....6.6d
0010   0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00    ........
0018   0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00    ........
0020   0x00 0x00 0x00 0x00 0x00 0x00 0x07 0x00    ........
0028   0x00 0x03 0x36 0x04 0x80 0x6c 0x04 0x80    ..6..l..
0030   0x05 0x0c 0x00 0xe8 0x00 0x00 0x00 0x00    ........
0038   0x03 0x36 0x03 0x36 0x64 0x00 0x00 0x00    .6.6d...
0040   0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00    ........
0048   0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00    ........
0050   0x00 0x00 0x00 0x07 0x00 0x00 0x03 0x36    .......6
0058   0x05 0x80 0x6c 0x05 0x80 0x05 0x0c 0x00    ..l.....
0060   0xe8 0x00 0x00 0x00 0x00 0x03 0x36 0x03    ......6.
0068   0x36 0x64 0x00 0x00 0x00 0x00 0x00 0x00    6d......
0070   0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00    ........
0078   0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00    ........
0080   0x07 0x00 0x00 0x03 0x36 0x06 0x80 0x6c    ....6..l
